### ISSUE TITLE
The Redux Framework layout and the end user.

### ISSUE THREAD

**Issue by paaljoachim** [reduxframework/redux-framework#854]
I am brainstorming below and will do a recap right here:
- Framework set together for end user. All options in place in the various modules logically placed together. The end user chooses from options layout 1,2,3 checking out how the layout of options and which options are in place selecting whatever they feel is most usable for them.
## 

Redux bring all these lego blocks (options). It is up to each developer how these are put as modules and which options are included in the framework each developer creates based on these options.

It would be nice if Redux could create a "most" used end user framework (based on logical needs or guesswork and ask the developers using Redux today if they want their enduser result showcased on the web site). 
Providing the first option in the framework for selecting layout options.

It is like providing the elements for building .... a house, motorcycle etc etc.
It would be awesome if Redux could show off their own finished house and motorcycle (or from other developers).

Have the first option in the framework, an option to use framework layout 1, 2, 3. Selecting an option would then change the framework around to show the house, motorcycle etc. The developer could create something else and save a new options layout for enduser. So end user could get another layout option to choose from option 4. These options will then setup how the enduser sees the options and modules in the framework.

Redux provides the library of resources but it is the final result that gets most attention from the non developers. As a enduser framework that can be used right away will also be easier to show and promote for use by anyone. A ps can then be added...mentioning that the framework is highly customizable. Giving the user the option to move, add, delete options inside the framework to save another framework layout that they find more practical to use. Meaning import and export of layout options.

This also means that other developers can send in how they put together their own Redux Framework they are using for their own end users. One could then go through the options layout page on the web site and download and import them into Redux plugin to have additional end user options layouts to use. The tricky part is if one begins using one setup and at a later time switches to another layout with other options visible.

The bottom line is this:
We can see end user frameworks here and there by themify.me, mysitemyway.com and others. Having Redux also make enduser frameworks as an additional promotion to show the flexibility and use will really boost Redux.
